packages/apputils-extension/src/announcements.ts

```diff
@@ -153,6 +153,16 @@ export const announcements: JupyterFrontEndPlugin<void> = {
             ]
           }
         );
+        const alertId = Notification.emit(
+          trans.__(
+            'For best user experience when using screenreader, turn off scan mode. Use the command: "Modifier Key + Spacebar." For ORCA use: "Insert+A"'
+          ),
+          'default',
+          {
+            autoClose: false
+          }
+        );
+        return alertId;
```

Comment:
I do not think that we should display this message to non-screenreader users, they would be (rightly) annoyed. I do not see logic making this conditional on screen reader use, so I assume this was intended to show to everyone or this PR is still work in progress? If it is the latter, I would suggest switching it to draft status to help reviewers prioritise the PRs which are ready.

Reply:
Hi @krassowski I have made some changes to the problem you mentioned above. 

- Firstly, the changes I have made should only notify the screen reader users that they should disable scan mode to make Jupyterlab screen reader friendly. 
- Aria live is set to` role: "assertive"` because that seemed to be the only way the screen reader would pick it up. 
- Furthermore, I have also placed the aria live append in the `announcement.ts` file as the scan mode information we are trying to convey is an announcement in itself.